{
  "uid" : "54d306154e71e379",
  "name" : "[US-444]/[10] I would like to see registration error, when I don't fill mobile phone input",
  "fullName" : "tests.WebAPP.RegistrationValidators_Tests.test_10",
  "historyId" : "e0044e262b3e592b97f8312bc403da9",
  "time" : {
    "start" : 1602694583658,
    "stop" : 1602694584700,
    "duration" : 1042
  },
  "description" : "[US-444]/[10] As a user I would like to see registration error, when I don't fill mobile phone input",
  "descriptionHtml" : "<p>[US-444]/[10] As a user I would like to see registration error, when I don't fill mobile phone input</p>\n",
  "status" : "failed",
  "statusMessage" : "Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]",
  "statusTrace" : "java.lang.AssertionError: Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]\n\tat org.testng.Assert.fail(Assert.java:99)\n\tat org.testng.Assert.failNotEquals(Assert.java:1037)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:140)\n\tat org.testng.Assert.assertEquals(Assert.java:122)\n\tat org.testng.Assert.assertEquals(Assert.java:629)\n\tat com.steps.MainPageSteps.iOpenHomePage(MainPageSteps.java:35)\n\tat tests.WebAPP.RegistrationValidators_Tests.test_10(RegistrationValidators_Tests.java:402)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)\n\tat org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\n\tat java.base/java.lang.Thread.run(Thread.java:832)\n",
  "flaky" : false,
  "newFailed" : true,
  "beforeStages" : [ {
    "name" : "Setting up Test Class",
    "time" : {
      "start" : 1602694582880,
      "stop" : 1602694583646,
      "duration" : 766
    },
    "description" : "Setting up Test Class",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "Setting up Excel File",
    "time" : {
      "start" : 1602694523548,
      "stop" : 1602694523599,
      "duration" : 51
    },
    "description" : "Setting up Excel File",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "description" : "[US-444]/[10] As a user I would like to see registration error, when I don't fill mobile phone input",
    "status" : "failed",
    "statusMessage" : "Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]",
    "statusTrace" : "java.lang.AssertionError: Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]\n\tat org.testng.Assert.fail(Assert.java:99)\n\tat org.testng.Assert.failNotEquals(Assert.java:1037)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:140)\n\tat org.testng.Assert.assertEquals(Assert.java:122)\n\tat org.testng.Assert.assertEquals(Assert.java:629)\n\tat com.steps.MainPageSteps.iOpenHomePage(MainPageSteps.java:35)\n\tat tests.WebAPP.RegistrationValidators_Tests.test_10(RegistrationValidators_Tests.java:402)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)\n\tat org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\n\tat java.base/java.lang.Thread.run(Thread.java:832)\n",
    "steps" : [ {
      "name" : "I open home page",
      "time" : {
        "start" : 1602694584102,
        "stop" : 1602694584212,
        "duration" : 110
      },
      "status" : "failed",
      "statusMessage" : "Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]",
      "statusTrace" : "java.lang.AssertionError: Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]\n\tat org.testng.Assert.fail(Assert.java:99)\n\tat org.testng.Assert.failNotEquals(Assert.java:1037)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:140)\n\tat org.testng.Assert.assertEquals(Assert.java:122)\n\tat org.testng.Assert.assertEquals(Assert.java:629)\n\tat com.steps.MainPageSteps.iOpenHomePage(MainPageSteps.java:35)\n\tat tests.WebAPP.RegistrationValidators_Tests.test_10(RegistrationValidators_Tests.java:402)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)\n\tat org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\n\tat java.base/java.lang.Thread.run(Thread.java:832)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ {
      "uid" : "2a221b3650d66392",
      "name" : "Scenario FAIL screenshot",
      "source" : "2a221b3650d66392.png",
      "type" : "image/png",
      "size" : 18343
    }, {
      "uid" : "43f57254cf7f84b8",
      "name" : "TestNG test FAIL logs",
      "source" : "43f57254cf7f84b8.txt",
      "type" : "text/plain",
      "size" : 2582
    } ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "Teardown Test Class",
    "time" : {
      "start" : 1602694584702,
      "stop" : 1602694584768,
      "duration" : 66
    },
    "description" : "Teardown Test Class",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "Teardown Test Suite",
    "time" : {
      "start" : 1602695039259,
      "stop" : 1602695040517,
      "duration" : 1258
    },
    "description" : "Teardown Test Suite",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "package",
    "value" : "tests.WebAPP.RegistrationValidators_Tests"
  }, {
    "name" : "testClass",
    "value" : "tests.WebAPP.RegistrationValidators_Tests"
  }, {
    "name" : "testMethod",
    "value" : "test_10"
  }, {
    "name" : "parentSuite",
    "value" : "Automationpractice.com WEB - Kamil Nowocin"
  }, {
    "name" : "suite",
    "value" : "Automated tests for WEB client"
  }, {
    "name" : "subSuite",
    "value" : "tests.WebAPP.RegistrationValidators_Tests"
  }, {
    "name" : "host",
    "value" : "travis-job-a8c8b075-3079-49fb-9f08-6d508dcf90e2"
  }, {
    "name" : "thread",
    "value" : "6396@travis-job-a8c8b075-3079-49fb-9f08-6d508dcf90e2.TestNG-test=Automated tests for WEB client-1(23)"
  }, {
    "name" : "framework",
    "value" : "testng"
  }, {
    "name" : "language",
    "value" : "java"
  }, {
    "name" : "story",
    "value" : "NEGATIVE FLOW"
  }, {
    "name" : "owner",
    "value" : "Kamil Nowocin"
  }, {
    "name" : "feature",
    "value" : "REGISTRATION VALIDATORS TESTS"
  }, {
    "name" : "epic",
    "value" : "Web App Tests"
  }, {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ {
    "name" : "TAP-0029",
    "url" : "https://tracker.FAKE.com/jira/browse/AUTOMATION_PRACTICE/issue/TAP-0029",
    "type" : "issue"
  }, {
    "name" : "STORY-444",
    "url" : "https://tracker.FAKE.com/jira/browse/AUTOMATION_PRACTICE/STORY-444",
    "type" : "tms"
  } ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "owner" : "Kamil Nowocin",
    "retries" : [ {
      "uid" : "bc47b983f2374c13",
      "status" : "skipped",
      "statusDetails" : "Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]",
      "time" : {
        "start" : 1602694577369,
        "stop" : 1602694582818,
        "duration" : 5449
      }
    } ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 3,
        "broken" : 7,
        "skipped" : 14,
        "passed" : 268,
        "unknown" : 0,
        "total" : 292
      },
      "items" : [ {
        "uid" : "89be8ba9c7c7c329",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/189766563/#testresult/89be8ba9c7c7c329",
        "status" : "passed",
        "time" : {
          "start" : 1602608142411,
          "stop" : 1602608173391,
          "duration" : 30980
        }
      }, {
        "uid" : "e1c90c21459aaa85",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/189536094/#testresult/e1c90c21459aaa85",
        "status" : "passed",
        "time" : {
          "start" : 1602521750180,
          "stop" : 1602521788009,
          "duration" : 37829
        }
      }, {
        "uid" : "6caec9f6b4dd6be5",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/189323165/#testresult/6caec9f6b4dd6be5",
        "status" : "passed",
        "time" : {
          "start" : 1602435282121,
          "stop" : 1602435315028,
          "duration" : 32907
        }
      }, {
        "uid" : "953d0cb11bf8a0cc",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/189207237/#testresult/953d0cb11bf8a0cc",
        "status" : "passed",
        "time" : {
          "start" : 1602348861941,
          "stop" : 1602348897273,
          "duration" : 35332
        }
      }, {
        "uid" : "89cd0fb4d4cee938",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/189083485/#testresult/89cd0fb4d4cee938",
        "status" : "passed",
        "time" : {
          "start" : 1602262429473,
          "stop" : 1602262461514,
          "duration" : 32041
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "54d306154e71e379.json",
  "parameterValues" : [ ]
}